README.md

```diff
@@ -352,3 +352,64 @@ Planned features:
   * Support for integer textures and arithmetic
 
   * Compute shaders
+
+
+Halide for Hexagon HVX
+======================
+Halide now supports the Qualcomm Hexagon DSP. The Hexagon DSP provides a set of 64 and 128 byte vector instructions - the Hexagon Vector eXtensions (HVX). HVX is well suited to image processing and Halide for Hexagon HVX will generate the appropriate HVX vector instructions from a program authored in Halide.
```

Comment:
This is all written as describing a *new* feature. This text will age poorly. It should be a statement about the world as it is, not a description of what's new.

E.g. Change first sentence to: Halide supports offloading work to Qualcomm Hexagon DSP